worker.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * @author Loreto Parisi (loretoparisi at gmail dot com)
```

Comment:
we avoid author tags in the source in our repo. github's history includes more detailed authorship information already. would you be ok removing this?